db/migrate/20231011213406_remove_apple_excluded_category.rb

```diff
@@ -0,0 +1,19 @@
+class RemoveAppleExcludedCategory < ActiveRecord::Migration[7.0]
+  def change
+    Apple::Config.all.each do |ac|
+      pub = ac.build_publisher
+
+      eps =
+        pub.show.podcast_episodes.filter do |ep|
+          next false unless ep.feeder_episode.categories.include?("apple-excluded")
+          ep.feeder_episode.update(categories: ep.feeder_episode.categories - ["apple-excluded"])
```

Comment:
Apple ID / GUID changes

This fixes up the existing episodes having remote episode ids as their guids. This removes the excluded category so that they end up in the set of episodes to sync (The original fix for this bad data was to exclude the episodes from `Apple::Publisher#episodes_to_sync`).